README.md

```diff
@@ -1,2 +1,105 @@
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+  [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb)
+
+The OracleDB library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This provides the functionality required to access and manipulate data stored in a OracleDB database.  
+
+For more information on the operations supported by the `oracledb:Client`, which include the below, go to [The OracleDB Package](https://ballerina.io/learn/api-docs/ballerina/oracledb/).
+
+- Pooling connections
+- Querying data
+- Inserting data
+- Updating data
+- Deleting data
+- Updating data in batches
+- Executing stored procedures
+- Closing the client
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/oracledb-init-options.html).
+
+## Issues and Projects 
+
+Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library). 
+
+This repository only contains the source code for the package.
+
+## Building from the Source
+
+### Setting Up the Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+   * [OpenJDK](http://openjdk.java.net/install/index.html)
+
+2. Download and install [Docker](https://www.docker.com/get-started)
+   
+3. Export Github Personal access token with read package permissions as follows,
+        
+        export packageUser=<Username>
+        export packagePAT=<Personal access token>
+
+### Building the Source
+
+Execute the commands below to build from the source.
+
+1. To build the library:
+
+        ./gradlew clean build
+        
+2. To run the integration tests:
+
+        ./gradlew clean test
+
+3. To build the package without tests:
+
+        ./gradlew clean build -x test
+
+4. To run only specific tests:
+
+        ./gradlew clean build -Pgroups=<Comma separated groups/test cases>
+
+   **Tip:** The following groups of test cases are available.<br>
+   Groups | Test Cases
+   ---| ---
+   connection | connection-init<br> ssl
+   pool | pool
+   transaction | local-transaction <br> xa-transaction
+   execute | execute-basic <br> execute-params
+   batch-execute | batch-execute 
+   query | query-simple-params<br>query-numeric-params<br>query-complex-params
+   procedures | procedures
```

Comment:
Are these valid groups?

Reply:
Changed to match with b04466c7e3261891042328ee057f4959563e158f